test-lib.sh

```diff
@@ -1327,7 +1364,6 @@ ct_run_tests_from_testset() {
     fi
     time_diff=$(ct_timestamp_diff "$time_beg" "$time_end")
     printf -v TEST_SUMMARY "%s %s for '%s' %s (%s)\n" "${TEST_SUMMARY:-}" "${test_msg}" "${app_name}" "$test_case" "$time_diff"
-    [ -n "${FAIL_QUICKLY:-}" ] && return 1
```

Comment:
It is related. Explanation:

This PR introduces checking of return value with which the trap on EXIT function is called, so that we could see if some weird (not expected) failures happen in the tests.
But if [ -n "${FAIL_QUICKLY}" ] is evaluated as not true (which is always, because we do not use it at all) and is also the last command of the shell (which is in except of ruby container tests always) then the shell ends with 1 and testsuite ends with FAILURE, as the return value is caught and evaluated by the trap function.

I counted with a fact, that this functionality is not used and that we would need to introduce something more robust during the `prepare` functions that we have in few of our containers' tests anyway. So I removed that.
However, if you think that this line should be kept in the code I will rework the solution.

Reply:
If we have a variable like `FAIL_QUICKLY` defined, but never used, I am fine with removal.